### PR TITLE
fix html page formatting

### PR DESCRIPTION
**Pull request by surajthapa12**
#### What's this PR do?
This PR updates the FAQ as requested.

#### Why are we doing this? How does it help us?
Customer request.

#### How should this be manually tested?
Regular local test procedures.

#### How should this change be communicated to end users?
Once live, we can communicate with end users.

#### Are there any smells or added technical debt to note?
Nope.

#### What are the relevant tickets?
[Airtable ticket](https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwgojVPEBjVPvt59/recn3NouNtA5aDLKH?blocks=hide)

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***

* [ ] Added automated tests? *( )*
* [ ] Tested manually on mobile? *( )*
* [ ] Checked BrowserStack? *( )*
* [ ] Checked for performance implications? *( )*
* [ ] Checked accessibility? *( )*
* [ ] Checked for security implications? *( )*
* [ ] Updated the documentation/wiki? *( )*

#### TODOs / next steps:

* [ ] *Schedule a time with one of the team members to be able to run it locally.*
* [ ] *Schedule a time to test it in staging server*

### CODE REVIEW

File: templates/includes/faq.html [hunk 1]
```diff
@@ -6,10 +6,21 @@
...
+
+      </p></div>
 
       <h4 class="grid_separator--s">How can I renew my membership?</h4>
-      <div class="prose grid_separator--l"><p>Sustaining memberships automatically renew on a monthly or yearly basis. If you need to update your billing information, or your membership is up for renewal soon, please contact <a href="mailto:membership@texastribune.org">membership@texastribune.org</a>.</p></div>
+      <div class="prose grid_separator--l">
+        <p>If you’re looking to renew your membership, make a donation to the Tribune. Your membership renews for one full year after you make a donation.</p>
+
+        <p>Sustaining memberships automatically renew on a monthly or yearly basis. If you need to update your credit card information, <a href="http://texastribune.org/login">log in</a> to your Tribune account. If you need assistance, please contact <a href="mailto:membership@texastribune.org">membership@texastribune.org</a>.</p>
```
Comment (ashley-hebler) [inline templates/includes/faq.html:22]: This url will auto redirect to https:// but we should reference `https://texastribune.org/login` instead for good measure. (I know that's probably just how it was in Airtable 😄)

**Review by ashley-hebler — COMMENTED** [review]
Left a few suggestions, but otherwise LGTM!

File: templates/includes/faq.html [hunk 2]
```diff
@@ -6,10 +6,21 @@
...
+      <div class="prose grid_separator--l"><p>
+        A sustaining membership means your donation recurs automatically on a monthly or yearly basis via credit card. This provides the Tribune with the sort of predictable revenue that allows us to plan budgets a few years at a time, and it helps eliminate renewal mailings or lapses in your membership.
+
+        </p> You can update the credit card information associated with your monthly or yearly donation on your Tribune account. <a href="http://texastribune.org/login">Log in</a>, then update your payment information from the “Membership” tab.</p>
+
+        </p>Don’t have a Tribune account? <a href="http://texastribune.org/signup">Sign up</a> to view your donation history and  membership status, update card information for recurring donations, and download your yearly tax receipt. <a href="https://www.texastribune.org/account-faq">Learn more about Tribune accounts. </a></p>
+
+      </p></div>
```
[the remark this answers is not in the document]
Comment (surajthapa12) [reply templates/includes/faq.html:16]: Thank you for the suggestions. It definitely looks better. I am implementing it. Thank you.

File: templates/includes/faq.html (hunk 1 above)
Comment (surajthapa12) [reply templates/includes/faq.html:25]: Thank you for this. I am assuming you meant for https://texastribune.org/signup . I am updating it to /login. Yep, I was following the airtable. Very good to know about this redirect.

Thank you for looking at it so closely.

**Pull request merged**